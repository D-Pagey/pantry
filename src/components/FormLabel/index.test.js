import React from 'react';
import { FormLabel } from '.';

describe('FormLabel component', () => {
    it('should render', () => {
        const { container } = render(<FormLabel>Label</FormLabel>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
