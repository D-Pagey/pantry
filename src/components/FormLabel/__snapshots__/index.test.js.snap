// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormLabel component should render 1`] = `
.c0 {
  padding: 0 0 0.5rem;
}

<label
  class="c0"
>
  Label
</label>
`;
